.github/workflows/rust.yml

```diff
@@ -49,6 +49,23 @@ jobs:
     - run: cargo clippy --all-targets --target ${{ matrix.target }}
     - run: cargo test --target ${{ matrix.target }}
 
+  e2e-test:
```

Comment:
FYI this test won't be required because it isn't included in the `complete` job's required jobs, which I think is good for now because this test will probably be brittle.